tests/cucumber/steps/steps.js

```diff
@@ -4421,6 +4446,18 @@ module.exports = function getSteps(options) {
     }
   );
 
+  Then('fee field is in txn', async function () {
+    const s = algosdk.decodeObj(this.stx);
+    const { txn } = s;
+    assert.strictEqual('fee' in txn, true);
```

Comment:
Not worth changing this over, but I thought I would mention that the `myObj.hasOwnProperty('key')` is preferred in JavaScript. The `in` operator will match keys across the object's entire prototype chain whereas `hasOwnProperty()` only checks the object's own keys. It's not a big deal at all in this case, but it can sometimes cause hard-to-catch bugs. JavaScript is weird 😄